modules/nw-create-load-balancer-service.adoc

```diff
@@ -38,15 +38,22 @@ spec:
   - name: db
     port: 3306 <2>
   loadBalancerIP:
-  type: LoadBalancer <3>
+  loadBalancerSourceRanges: 10.0.0.0/8,192.168.0.0/16 <3>
```

Comment:
This needs to be a yaml list:
```suggestion
  loadBalancerSourceRanges: <3>
  - 10.0.0.0/8
  - 192.168.0.0/16
```

Reply:
I have addressed it in the latest commit.